aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -1899,12 +1894,12 @@ static Tensor & masked_select_out_impl_cpu(Tensor & result, const Tensor & self,
   auto mask_long_data = mask_long.data_ptr<int64_t>();
   auto mask_prefix_sum_data = mask_prefix_sum.data_ptr<int64_t>();
   // TODO: Here can only use std::partial_sum for C++14,
-  // use std::exclusive_scan when PyTorch upgrades to C++17, which have better peformance.
+  // use std::exclusive_scan when PyTorch upgrades to C++17, which have better performance.
   // std::exclusive_scan(mask_long_data, mask_long_data + mask_long.numel(), mask_prefix_sum_data, 0);
```

Comment:
@malfet can you swap out the comments to std::exclusive_scan now that we are in C++17?

Reply:
I'll do it (and few other changes) in followup PR (don't want to muddy this one with unrelated functional changes)

Reply:
https://github.com/pytorch/pytorch/pull/96217